### DISCUSSION TITLE
Suggestion: replace dependency

### DISCUSSION THREAD

**Discussion by Semigradsky** [eslint/eslint#15972] [Ideas]
Replace [`json-stable-stringify-without-jsonify`](https://github.com/samn/json-stable-stringify) to [`fast-json-stable-stringify`](https://github.com/epoberezkin/fast-json-stable-stringify).
It is totally the same but slightly faster (without `Array.isArray` and `Object.keys` polyfills in code).

`fast-json-stable-stringify` used by `ts-jest`, `@jest/transform`, ...

**Comment by bmish** [comment]
Sounds like a reasonable suggestion given that they are both very popular:

- https://www.npmjs.com/package/json-stable-stringify-without-jsonify
- https://www.npmjs.com/package/fast-json-stable-stringify

This might be better located as an issue instead of discussion though.

**Reply by Semigradsky** [reply]
https://github.com/eslint/eslint/issues/15973

**Reply by Semigradsky** [reply]
`json-stable-stringify-without-jsonify` is current dependency, so of course it has many downloads :-)
